core/src/main/java/nl/nn/adapterframework/receivers/Receiver.java

```diff
@@ -1217,27 +1213,15 @@ private Message processMessageInAdapter(MessageWrapper<M> messageWrapper, PipeLi
 			Message result = null;
 			PipeLineResult pipeLineResult = null;
 			try {
-				Message pipelineMessage;
-				if (getListener() instanceof IBulkDataListener) {
```

Comment:
Ik dacht dat we IBulkDataListener pas in 8.0 zouden verwijderen, niet in 7.9.x series, hoewel het niet waarschijnlijk is dat iemand dat gebruikt...

Reply:
Tja, klopt maar ik zag geen implementaties, en het stoorde mij een beetje ....

Reply:
Ik vind het ook erg storend, implementaties zouden in custom code moeten zitten maar ik heb geen idee of er ergens enige custom-code is die die interface implementeert.

Maar als je dit stukje weghaalt, dan kan misschien ook die interface weg? Die wordt verder helemaal nergens gerefereerd, bij mijn weten.

Reply:
Done!